Rtp/RtpClient.cs

```diff
@@ -3382,7 +3382,7 @@ protected internal void OnOutOfBandData(byte[] data, int offset, int length)
 
             if (m_ThreadEvents)
             {
-                m_EventData.Enqueue(new Tuple<TransportContext, Common.BaseDisposable, bool, bool>(null, new Common.Classes.PacketBase(data, offset, length, true, true), true, true));
+                m_EventData.Enqueue((Context: null, Packet: new Common.Classes.PacketBase(data, offset, length, true, true), Final: true));
```

Comment:
Add SentPacket here, same as Final... ```Sent: False``` for sent packets. I don't think there is any logic which enqueues for sent packets, it was a bug for these to be `true` always.

Thanks for finding this!